Taarafo.Portal.Web/Program.cs

```diff
@@ -17,9 +18,24 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
+            .ConfigureAppConfiguration((hostingContext, config) =>
+            {
+                var environment = hostingContext.HostingEnvironment;
+                config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+
+                if (!string.IsNullOrEmpty(environment.EnvironmentName))
+                {
+                    config.AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true);
+                };
+
+                if (hostingContext.HostingEnvironment.IsDevelopment())
                 {
-                    webBuilder.UseStartup<Startup>();
-                });
+                    config.AddUserSecrets<Program>();
+                }
+            })
```

Comment:
Ideally we'd be using a KeyVault to store the secrets and just bring those in.
For now, I think just adding the key in the appSettings.Development.json should be enough. Which is what you did in the code below.
Let's stick to that and remove this new configuration for app for now.

Reply:
@hassanhabib yes, my thoughts was user secrets locally in development and then KeyVault for production, then the keys is not checked in.  Or are you suggesting we generate a code on Syncfusion for development and have that checked in?

Reply:
@cjdutoit that's correct, you would need to generate a community license key and just keep it in the `appSettings.Development.json` file. this file is supposed to be ignored. never pushed to our master branch (or any other branch for that matter).

Reply:
@hassanhabib  OK, I will revert the change.  User-secrets helpful here as its always local.